src/routes/events/events.service.spec.ts

```diff
@@ -33,6 +33,10 @@ describe('EventsService', () => {
     eventsService = module.get<EventsService>(EventsService);
   });
 
+  afterEach(async () => {
+    jest.clearAllMocks()
```

Comment:
Why is this required?

Reply:
because this line was not passing https://github.com/safe-global/safe-events-service/pull/66/files#diff-d79512aca03ad68a51d5b4e98066338dcb424638d13712a3f0b9d909ba23ee63R80 
looks like mock keep the number of calls between tests.

Reply:
This does not sound right, as we are using mocks for same functions and I didn't have to do that, like https://github.com/safe-global/safe-events-service/blob/b112382582569a6a10326b6f41a55ab77bdab43f/src/routes/webhook/webhook.service.spec.ts#L23

Reply:
I did it at different way, the issue was that the mock `webhookService`  was declared globally out of `beforeEach` and the `spyOn`  uses this object that was declared globally, consequently the calls are accumulated between tests.